### PR TITLE
Fix badge main on small screen

### PR DESCRIPTION
**Pull request by smyzin**
- added a new breakpoint
- added custom styles
- fixed badge name

### CODE REVIEW

File: src/hooks/useBreakpoints.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export default function () {
   }, [])
 
   return {
+    badgeNameSize: width > smSize && width < 400,
```
Comment (T-Damer) [inline src/hooks/useBreakpoints.ts:23]: The `useBreakpoint` is used to have different screens breakpoints, maybe you can make a different name for this size?

File: src/index.css [hunk 2]
```diff
@@ -121,6 +121,16 @@ body {
   }
 }
 
+.badgeBlockName {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
```
Comment (T-Damer) [inline src/index.css:127]: The docs say, that you can use `flex-direction`: https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient

File: src/index.css [hunk 3]
```diff
@@ -121,6 +121,16 @@ body {
   }
 }
 
+.badgeBlockName {
+  display: -webkit-box;
+  -webkit-line-clamp: 2;
```
Comment (T-Damer) [inline src/index.css:126]: What about `Ff`? The `webkit` won't work with it

Comment (T-Damer) [reply src/index.css:126]: We can use `tailwind` plugin for this: https://tailwindcss.com/docs/plugins#line-clamp

File: src/index.css [hunk 4]
```diff
@@ -121,6 +121,16 @@ body {
   }
 }
 
+.badgeBlockName {
+  display: -webkit-box;
```
Comment (T-Damer) [inline src/index.css:125]: If we can switch to `flex` model, this would be great. If we can't — then let's use this class 👍

**Review by T-Damer — CHANGES_REQUESTED** [review]
Gave comments

File: src/index.css (hunk 3 above)
Comment (smyzin) [reply src/index.css:126]: Set plugin to project

File: src/index.css [hunk 5]
```diff
@@ -121,6 +121,16 @@ body {
...
+  -webkit-line-clamp: 2;
+  -webkit-box-orient: vertical;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 6.25rem;
+  height: 2.75rem;
+}
```
[the remark this answers is not in the document]
Comment (smyzin) [reply src/index.css:132]: Made it via tailwind

File: src/index.css (hunk 2 above)
Comment (smyzin) [reply src/index.css:127]: Set plugin instead

**Review by T-Damer — APPROVED** [review]
Great work :godmode:

**Pull request merged**